pkg/main.go

```diff
@@ -1,23 +1,49 @@
 package main
 
 import (
-	"github.com/sarulabs/dingo"
-	"github.com/softonic/homing-pigeon/pkg/generatedServices/dic"
+	"github.com/softonic/homing-pigeon/pkg/ack"
+	"github.com/softonic/homing-pigeon/pkg/messages"
+	"github.com/softonic/homing-pigeon/pkg/middleware"
+	"github.com/softonic/homing-pigeon/pkg/readers"
+	"github.com/softonic/homing-pigeon/pkg/writers"
 	"k8s.io/klog"
+	"os"
+	"strconv"
 )
 
 func main() {
 	klog.InitFlags(nil)
 
-	container, err := dic.NewContainer(dingo.App)
+	bufLen, err := strconv.Atoi(os.Getenv("MESSAGE_BUFFER_LENGTH"))
+	if err != nil {
+		bufLen = 0
+	}
+	inputChannel := make(chan messages.Message, bufLen)
+	outputChannel := make(chan messages.Message, bufLen)
+
+	bufLen, err = strconv.Atoi(os.Getenv("ACK_BUFFER_LENGTH"))
+	if err != nil {
+		bufLen = 0
+	}
+	ackChannel := make(chan messages.Ack, bufLen)
+	brokerAckChannel := make(chan messages.Ack, bufLen)
+	ackManagerChannel := make(chan messages.Ack, bufLen)
```

Comment:
How about preparing the way of the return leg for this request-response pattern?
We could rename `ackChannel` to `responseChannel` or something, and `ackMangaerChannel` to `responseAsyncMiddlewareChannel` or something?